compose/ui/ui/src/uikitMain/kotlin/androidx/compose/ui/interop/ComposeUITextField.uikit.kt

```diff
@@ -38,7 +40,9 @@ fun ComposeUITextField(value: String, onValueChange: (String) -> Unit, modifier:
         },
         modifier = modifier,
         update = { textField ->
-            textField.text = value
+            interopContext.deferAction {
+                textField.text = value
+            }
```

Comment:
You used interopContext.deferAction inside `update`
But maybe we can wrap the whole update in interopContext.deferAction as well ? So users will not need it. (And we can keep LocalUIKitInteropContext internal)

Reply:
Nice idea, implemented it.

Reply:
https://github.com/JetBrains/compose-multiplatform-core/pull/773/commits/c72351869c27bc96c8cf89a24477d92f48545471